components/widgets/menu/menu_items/useWords.tsx

```diff
@@ -0,0 +1,162 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {useIntl} from 'react-intl';
+import {PrimitiveType, FormatXMLElementFn} from 'intl-messageformat';
+
+import {limitThresholds, asGBString, inK} from 'utils/limits';
+import {t} from 'utils/i18n';
+
+import useOpenPricingModal from 'components/common/hooks/useOpenPricingModal';
+import {LimitTypes, LimitSummary} from 'components/common/hooks/useGetHighestThresholdCloudLimit';
+
+interface Words {
+    title: React.ReactNode;
+    description: React.ReactNode;
+    status: React.ReactNode;
+}
+
+export default function useWords(highestLimit: LimitSummary | false, isAdminUser: boolean): Words | false {
+    const intl = useIntl();
+    const openPricingModal = useOpenPricingModal();
+    if (!highestLimit) {
+        return false;
+    }
+    const usageRatio = (highestLimit.usage / highestLimit.limit) * 100;
+    let callToAction = intl.formatMessage({
+        id: 'workspace_limits.menu_limit.view_plans',
+        defaultMessage: 'View plans',
+    });
```

Comment:
Can we remove this link until https://mattermost.atlassian.net/browse/MM-43648 is complete? Opening the pricing modal as is to end users still shows all upgrade and trial options

Reply:
The easiest thing would be to do a css hack and `display: none` the link for end users. That way it isn't in the DOM. Would that work?

If we try conditionally not rendering, we have to either
* break the pattern of placing `<a></a>` in the translation string.
* Maintain a separate list of end user strings.

Reply:
Yes, that would work. It's just while the other PR comes in. We'll make a note in that ticket to restore